sys-apps/polkit/polkit.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+
+import os
+import stdlib
+import stdlib.split.drain_all
+from stdlib.template import autotools
+from stdlib.template.configure import configure
+from stdlib.patch import patch
+from stdlib.manifest import manifest
+
+
+def patch_polkit():
+    # There should be a dedicated user and group to take control of the polkitd daemon after it's started
+    stdlib.cmd("groupadd -fg 27 polkitd &&                          \
```

Comment:
That will install the daemon on the host system (the one building the package), not the user's system.

You can install it on the user's system with an instruction file, which executes command after the package is installed. See [`this package`](https://github.com/raven-os/nbuild-manifests/tree/master/dev-libs/gtk3) for an example. The file `instructions.sh` contains commands to execute at different point of the installation. You want to fill the `after_install` function.

Remember to load the instruction file with `packages['xxx/yyy'].load_instructions('./instructions.sh')` at the end of the manifest and to put the both files in another subdirectory.